bin/yarn

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
```

Comment:
The presence of this file automatically adds `yarn install` to the asset precompilation: https://github.com/rails/rails/blob/v5.1.2/railties/lib/rails/tasks/yarn.rake#L9-L11